src/main/java/com/company/Main.java

```diff
@@ -14,23 +16,39 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
 import java.util.concurrent.*;
 
 public class Main {
 
     public static void main(String[] args) throws Exception {
 
+        /*
+        I left objects and methods that is used for file handling in case of adding new matrixes to DB
```

Comment:
use "//"
do not write what you did, write what code doing, like, create objects, create matrix, move something etc.